Update readme (#2)
DevExpress-Examples/xaf-how-to-show-audit-entries-for-a-current-object-and-its-aggregated-objects-in-one-list: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E4565)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->
<!-- default file list -->

<!-- default file list end -->

> # How to show audit entries for a current object and its aggregated objects in one list

Reviewer: ```suggestion
# XAF - How to show audit entries for a current object and its aggregated objects in one list
```

> This example shows how to show audit information about a current object and each object from its collection property:

Reviewer: ```suggestion
This example demonstrated how to show audit information about a current object and each object in its collection property:
```
